### PR TITLE
✨📝 create contact page html file #70

### PR DESCRIPTION
**Pull request by Schola-droid**
# Description

Create Contact Page HTML File

Fixes #81 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
Screenshot 2023-10-03 at 14 22 34
@https://github.com/Johnsonnjichi

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 
+
```
Comment (paulkahura) [inline index.html]: extra space

**Review by paulkahura — CHANGES_REQUESTED** [review]
Good attempt. check comment

File: index.html (hunk 1 above)
Comment (Schola-droid) [reply index.html]: > extra space

I've corrected it, Sir.

**Comment by JAPHETHNYARANGA** [comment]
Good attempt but please pick another isuue. Closing this pr as the issue is done already in another pr.

**Pull request closed without merging**